packages/theme/src/cli/services/rename.ts

```diff
@@ -3,15 +3,19 @@ import {themeComponent} from '../utilities/theme-ui.js'
 import {themeUpdate} from '@shopify/cli-kit/node/themes/api'
 import {AdminSession} from '@shopify/cli-kit/node/session'
 import {renderSuccess} from '@shopify/cli-kit/node/ui'
+import {promptThemeName} from '@shopify/cli-kit/node/themes/utils'
 
 export interface RenameOptions {
-  newName: string
+  name?: string
   development?: boolean
   theme?: string
   live?: boolean
+  environment?: string
 }
 
-export async function renameTheme(adminSession: AdminSession, options: RenameOptions) {
+export async function renameTheme(options: RenameOptions, adminSession: AdminSession) {
+  const newName = options.name || (await promptThemeName('New name for the theme'))
```

Comment:
ESLint is warning to change to `??` here (does pass linting though). I think it still makes sense to keep it as `||` in case of empty strings